src/seml/commands/manage.py

```diff
@@ -770,6 +770,7 @@ def reload_sources(
     from importlib.metadata import version
 
     from bson import ObjectId
+    from deepdiff import DeepDiff
```

Comment:
We could delay that to the dup detection code, maybe saves import runtime?

Reply:
We usually only delay packages to avoid them being imported for auto completion, I wouldn't bother here and be consistent with the rest of the code and only import at the start of methods